src/contracts/BPool.sol

```diff
@@ -669,4 +679,14 @@ contract BPool is BToken, BMath, IBPool {
   function _burnPoolShare(uint256 amount) internal {
     _burn(address(this), amount);
   }
+
+  /**
+   * @notice gets the value of the transient storage slot used for reentrancy locks
+   * @return _value its value
```

Comment:
bruh

Reply:
lmk if the extra info should be in a`@dev` entry instead

Reply:
@dev betta'